### DISCUSSION TITLE
Support RON for configuration

### DISCUSSION THREAD

**Discussion by bwidawsk** [JakeStanger/ironbar#670] [Ideas]
RON is a nice improvement over JSON while providing for a nice feel for rustaceans

From [ron documentation](https://github.com/ron-rs/ron)

> Note the following advantages of RON over JSON:
> 
>     trailing commas allowed
>     single- and multi-line comments
>     field names aren't quoted, so it's less verbose
>     optional struct names improve readability
>     enums are supported (and less verbose than their JSON representation)
>

**Comment by JakeStanger** [comment]
RON is actually (sort of) supported, it's just not documented because there were some potential limitations with its serde implementation that make it unsuitable and I never got round to testing. You can try creating a `.config/ironbar/config.ron` file and you might have some luck.
